script/bootstrap

```diff
@@ -76,7 +76,7 @@ bootstrap_carthage ()
   then
     carthage_cmd="$carthage_cmd --no-use-binaries"
   fi
-
+  echo "Disabled cache: $DISABLE_CARTHAGE_CACHE"
```

Comment:
Lets revert this commit. I'm gonna release a version of CarthageCache with better logging output.